### ISSUE TITLE
How to show the heatmap using trianed models?

### ISSUE THREAD

**Issue by MdAshrafulAlam** [yingcong/Facelet_Bank#11]
Has anyone used trained model to get the heat map? I want to get heat map images by using trained model inside the facelet_bank.

**Comment by yingcong** [comment]
Hi @MdAshrafulAlam 
Visualizing of the heatmap is as simple as follows:

Let w1 denote the attribute vector of one certain layer whose shape is nSamples x nChannels x Height x Width. More specifically, it is one element of "w" in
https://github.com/yingcong/Facelet_Bank/blob/0f2edb49ea72b4da5da6a12d08cbcd4ff7811f8f/test_facelet_net.py#L30

we can use
cv2.imwrite('heatmap1.jpg', torch.sum(w1[n]*w1[n], dim=0))
to visualize the heatmap of the n-th sample.
